app/lib/services/voip/MediaCallEvents.ios.test.ts

```
@@ -1,62 +1,49 @@
+		registerVoipToken: jest.fn(),
+		getInitialEvents: jest.fn(() => null),
+		clearInitialEvents: jest.fn(),
+		getLastVoipToken: jest.fn(() => ''),
+		stopNativeDDPClient: jest.fn(),
+		stopVoipCallService: jest.fn(),
+		addListener: jest.fn(),
+		removeListeners: jest.fn()
 	}
 }));
 
+import type { VoipPayload } from '../../../definitions/Voip';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)